### PR TITLE
Update Reporting_RabbitMQ_For_Kubernetes_Issues.md

### PR DESCRIPTION
**Pull request by pstack2021**
## Assessment of priority

Please delete the answers to the questions below as appropriate. 

For example, for the question: **How blocking is this documentation?:** The acceptable answers are: **Very/Somewhat/Not at all** if your answer is **Very** please delete the remaining answers **Somewhat** and **Not at all**.

**How blocking is this documentation?:** Very/Somewhat/Not at all

**Is there a user waiting on this change?:** Y/N

**Roughly how urgently is this documentation needed:** ASAP/This Week/This Month/At some point

### CODE REVIEW

File: docs/Reporting_RabbitMQ_For_Kubernetes_Issues.md [hunk 1]
```diff
@@ -1,33 +1,42 @@
...
+5. **What is the Kubernetes version?**
+   Run the following command to get the Kubernetes version: 
+   
+   ```
+   kubectl version
+   ``` 
+6. **Is the issue is related to an Installation/Upgrade procedure?**
+   If so, see "Specific issues during Installation/Upgrade of the commercial version" below. 
+
+7.
```
Comment (michaelklishin) [inline docs/Reporting_RabbitMQ_For_Kubernetes_Issues.md]: This empty line breaks list item numbering.

In general, nested lists is something to be avoided in Markdown. I'd suggest turning this list with **bold headers** into a set of sections 

``` md
#### Like This

#### and This

#### that each

 * Can
 * have
 * its own
 * nested
 * list
```

**Review by michaelklishin — CHANGES_REQUESTED** [review]

**Comment by pstack2021** [comment]
@DanielePalaia please review my changes to ensure they are technically correct before merging into the original document that you sent me for review. 
Many thanks,
Paula.

**Review by DanielePalaia — APPROVED** [review]

**Comment by DanielePalaia** [comment]
@pstack2021 yes fine for me, I will anyway review it again before sending to support! Thanks!

**Comment by michaelklishin** [comment]
I will take care of [this formatting suggestion](https://github.com/rabbitmq/support-tools/pull/34#discussion_r923317469).

**Comment by pstack2021** [comment]
thank you @michaelklishin

**Pull request merged**